spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/java/com/example/SecretManagerWebController.java

```diff
@@ -33,23 +31,26 @@
 @Controller
 public class SecretManagerWebController {
 
-  @Autowired private Environment environment;
+  private final SecretManagerTemplate secretManagerTemplate;
+  // Application secrets can be accessed using configuration properties class,
+  // secret can be refreshed when decorated with @RefreshScope on the class.
+  private final SecretConfiguration configuration;
 
-  @Autowired private SecretManagerTemplate secretManagerTemplate;
-
-  // Application secrets can be accessed using @Value and using the "sm://" syntax.
+  // Application secrets can be accessed using @Value syntax.
   @Value("${sm://application-secret}")
   private String appSecret;
 
-  // Multiple ways of loading the application-secret are demonstrated in bootstrap.properties.
-  // Try it with my-app-secret-1 or my-app-secret-2
-  @Value("${my-app-secret-1}")
-  private String myAppSecret;
+  public SecretManagerWebController(SecretManagerTemplate secretManagerTemplate,
+      SecretConfiguration configuration
+  ) {
+    this.secretManagerTemplate = secretManagerTemplate;
+    this.configuration = configuration;
+  }
 
   @GetMapping("/")
   public ModelAndView renderIndex(ModelMap map) {
-    map.put("applicationSecret", this.appSecret);
-    map.put("myApplicationSecret", this.myAppSecret);
+    map.put("applicationSecret", appSecret);
+    map.put("myApplicationSecret", configuration.getSecret());
```

Comment:
Rename those to match the new names that you'll be using in index.html

Reply:
done.